### ISSUE TITLE
GraphQL rate limiting plans?

### ISSUE THREAD

**Issue by moloney** [edgedb/edgedb#718]
Exposing GraphQL directly to clients is appealing (once the planned authentication/authorization pieces are in place) but I see no mention in the roadmap for rate limiting.  I understand this is also a bit more complicated for GraphQL vs REST since incoming queries can have essentially arbitrary complexity.  Are there plans for functionality like this?  Can you put it on your otherwise quite comprehensive roadmap?

**Comment by elprans** [comment]
Yes.  We are working on a comprehensive mechanism to control the QoS for the exposed ports.  For any interface, not just GraphQL.
